common/config/rush/command-line.json

```diff
@@ -180,7 +228,15 @@
 
       // For non-phased commands, we will continue to use the "associatedCommands" setting to associate
       // parameters with them.  (Phased commands must NOT appear in this list.)
-      "associatedCommands": ["build"]
+      "associatedCommands": ["build"],
+
+      // Phased commands expose parameters that are associated with their underlying phases.
+      // For example, "rush build" will accept the union of all parameters associated with any of
+      // its individual phases.
+      //
+      // The "--production" flag will be appended to package.json scripts for these specific phases,

```

Comment:
The `"node scripts/readme_updater.js"` script should print this CLI parameter value to confirm that it was passed along correctly.